src/components/Header.jsx

```diff
@@ -42,6 +46,9 @@ function Header() {
   const classes = useStyles();
   return (
     <Container>
+      <Alert className={classes.banner} severity="info">
+        Website Actively Under-Construction!
```

Comment:
How about surrounding this with 🚧  emojis?

```suggestion
        🚧 Website Actively Under-Construction! 🚧
```

Reply:
Also maybe we can make this the description and have an [**AlertTitle**](https://material-ui.com/components/alert/#description) that says "Warning"?

Reply:
One more thing. You could add an [Action](https://material-ui.com/components/alert/#actions) that links to the [subscription form URL](https://github.com/beyondhb1079/s4us/blob/d9ba90278b682b6f5ac037d21283ec6a8be8b496/src/config/constants.js#L3), e.g. `SUBSCRIBE FOR UPDATES`.